lld/ELF/LTO.cpp

```diff
@@ -186,6 +187,16 @@ BitcodeCompiler::BitcodeCompiler(Ctx &ctx) : ctx(ctx) {
         std::string(ctx.arg.thinLTOPrefixReplaceNew),
         std::string(ctx.arg.thinLTOPrefixReplaceNativeObject),
         ctx.arg.thinLTOEmitImportsFiles, indexFile.get(), onIndexWrite);
+  } else if (!ctx.arg.DTLTODistributor.empty() && !ctx.bitcodeFiles.empty()) {
+    StringRef version = getenv("LLD_VERSION"); // For testing only.
```

Comment:
a bit vague why this getenv is needed

Reply:
We thought it might be useful to pass this through to the distributor (via the JSON). This could allow incompatible distributors to report a diagnostic. Whilst this isn't useful immediately it might allow better diagnostics if the implementation in LLVM changed in some incompatible way that forced the distributors to be rewritten. We don't need this in our downstream implementation, so I haven't really tested the utility. I'm happy to remove this version stuff - at least for the initial support? However, I think some sort of compatibility check may eventually be important.

Reply:
I have removed the version stuff in this initial PR.